gprofiler/main.py

```diff
@@ -143,6 +143,7 @@ def __init__(
             self._system_metrics_monitor: SystemMetricsMonitorBase = SystemMetricsMonitor(self._stop_event)
         else:
             self._system_metrics_monitor = NoopSystemMetricsMonitor()
+        self._profiling_mode = cast(str, user_args["profiling_mode"])
```

Comment:
`user_args` are passed as a dict to be included in the metadata - it's nicer if you extract `profiling_mode` like all other args and pass it explicitly to `GProfiler`

Reply:
done